src/components/IngredientFeed.vue

```diff
@@ -82,18 +106,33 @@
     </div>
     <div class="row">
       <ul>
-        <li v-for="list in data" :key="list">
-          <div v-for="(types, name) in list" :key="name">
-            <div :id="name" class="row">
-              <h3>{{ name }}</h3>
+        <li>
```

Comment:
If no results in filteredItems:
  if !searchString.length:
     Display a header saying `Nothing in inventory, try adding some items`
  else:
     Display a header saying `No items matching <searchString>`